pkg/volume/flexvolume/mounter.go

```diff
@@ -47,8 +47,8 @@ func (f *flexVolumeMounter) SetUp(fsGroup *int64) error {
 // SetUpAt creates new directory.
 func (f *flexVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 	call := NewDriverCall(f.plugin.getExecutable(), mountCmd, exec.New())
-	call.AppendSpec(f.spec, fsGroup, f.plugin.host)
 	call.Append(dir)
+	call.AppendSpec(f.spec, fsGroup, f.plugin.host)
```

Comment:
Let keep it consistent between both the calls.. this is a nit.. I am fine with interface order..
Change this one to interface order too.. existing drivers have to be fixed anyways.